spec/features/admin/admin_can_view_order_details_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.